lib/davinci_dtr_test_kit/ext/inferno_core/runnable.rb

```diff
@@ -19,17 +17,6 @@ def allow_cors(*paths)
           route(:options, path, PRE_FLIGHT_HANDLER)
```

Comment:
Yeah I guess I just put it there because that was the tidiest way to do it at the time, but now that it's the only thing in the monkey patch, I'll move it

Reply:
https://github.com/inferno-framework/davinci-dtr-test-kit/pull/37/commits/874382d5e47defbb2803c581abda0c923d72fa92